### ISSUE TITLE
Node.js build Error in Vue.js Project with vue-tsc and Node 18.20.3

### ISSUE THREAD

**Issue by systemdfan** [marxunion/levach#1]
When trying to install lv4, a problem arose. Frontend is not building and nodejs container is not working. The logs contain the following:

```
> vue-tsc && vite build


/var/www/levach/frontend/node_modules/vue-tsc/bin/vue-tsc.js:68
                        throw err;
                        ^
Search string not found: "for (const existingRoot of buildInfoVersionMap.roots) {"
(Use `node --trace-uncaught ...` to show where the exception was thrown)
```

**Comment by lorkiofficial** [comment]
Hi.This problem is most likely due to the update of typescript to version 5.5(https://github.com/vuejs/language-tools/issues/4484), in the new commit I specified to use the old version 5.4, please update and enter the following commands:
```sh
cd frontend
rm -rf node_modules
npm cache clean --force
```
If problem saved, please report back, I will look for another solution

**Comment by lorkiofficial** [comment]
And also i'm updated vue-tsc to version 2.0.22, maybe problem is related to that

**Comment by LTo666** [comment]
> Hi.This problem is most likely due to the update of typescript to version 5.5([vuejs/language-tools#4484](https://github.com/vuejs/language-tools/issues/4484)), in the new commit I specified to use the old version 5.4, please update and enter the following commands:
> 
> ```shell
> cd frontend
> rm -rf node_modules
> npm cache clean --force
> ```
> 
> If problem saved, please report back, I will look for another solution

I did the same thing as you mentioned and it works.

Detail: https://github.com/fi3ework/vite-plugin-checker/issues/306#issuecomment-2184081185

**Comment by lorkiofficial** [comment]
> > Hi.This problem is most likely due to the update of typescript to version 5.5([vuejs/language-tools#4484](https://github.com/vuejs/language-tools/issues/4484)), in the new commit I specified to use the old version 5.4, please update and enter the following commands:
> > ```shell
> > cd frontend
> > rm -rf node_modules
> > npm cache clean --force
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > If problem saved, please report back, I will look for another solution
> 
> I did the same thing as you mentioned and it works.
> 
> Detail: [fi3ework/vite-plugin-checker#306 (comment)](https://github.com/fi3ework/vite-plugin-checker/issues/306#issuecomment-2184081185)

It's really good!
By the way I understand you used vue-tsc 1.0 version?

**Comment by systemdfan** [comment]
@LorkiVG thanks, the problem is gone now!

**Comment by systemdfan** [comment]
@LorkiVG thanks, the problem is gone now.

**Comment by LTo666** [comment]
> > > Hi.This problem is most likely due to the update of typescript to version 5.5([vuejs/language-tools#4484](https://github.com/vuejs/language-tools/issues/4484)), in the new commit I specified to use the old version 5.4, please update and enter the following commands:
> > > ```shell
> > > cd frontend
> > > rm -rf node_modules
> > > npm cache clean --force
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > If problem saved, please report back, I will look for another solution
> > 
> > 
> > I did the same thing as you mentioned and it works.
> > Detail: [fi3ework/vite-plugin-checker#306 (comment)](https://github.com/fi3ework/vite-plugin-checker/issues/306#issuecomment-2184081185)
> 
> It's really good! By the way I understand you used vue-tsc 1.0 version?

I use vue-tsc@1.8.27 in my local development environment, it works fine, but I don't know what version it used when it build the application in github action, I guess it should be the latest vue-tsc@1

**Comment by smzapp** [comment]
Just in case somebody will encounter this issue, I just installed the latest vue-tsc and it works. The current version is:

` "vue-tsc": "^2.2.0"`

`npm i vue-tsc@latest`

**Comment by zksdn** [comment]
@smzapp Thank you, the problem is solved after upgrading vue-tsc to version 2.2.0
